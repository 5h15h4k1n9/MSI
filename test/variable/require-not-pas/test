add(-10)
requireNonNegative()
exit
