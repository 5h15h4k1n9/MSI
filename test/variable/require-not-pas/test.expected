(void) void
Reverted with: Negative count not allowed
Bye-bye!
